reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -295,7 +295,7 @@ class StaticHtmlReporter : Reporter {
                 }
             }
             td { +ruleViolation.violation.rule }
-            td { +ruleViolation.violation.pkg.toCoordinates() }
+            td { +ruleViolation.violation.pkg?.toCoordinates().orEmpty() }
```

Comment:
Consider `?: "-"` instead of `orEmpty()` to be consistent with the next column.

Reply:
Good catch!